plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/PostgreSqlQueryRunner.java

```diff
@@ -92,6 +93,9 @@ public static void main(String[] args)
                 ImmutableMap.of(),
                 TpchTable.getTables());
 
+        queryRunner.installPlugin(new JmxPlugin());
+        queryRunner.createCatalog("jmx", "jmx");
```

Comment:
Using this in tests is still TODO, right?

Reply:
I see, thanks. A comment to that effect would be nice, though

Reply:
I think it is well known what is the need of JMX connector. I don't want to overcomment here.